### DISCUSSION TITLE
TRAIN ERROR:  map_location=torch.device('cpu')

### DISCUSSION THREAD

**Discussion by AgustinSaturni** [Project-MONAI/MONAILabel#1721] [Q&A]
Hi everyone Im traying to train a new radiology app to recognize the spleen. So Im using the radiology app with the deepedit model config. Im alredy labeled a few CTs and I want to train the model but Im using a notebook with an integrated gpu and I would like to use my CPU to go throug all the proces. Is it possible to config that in somewhere of the MonaiLabel's files?
I installed monailabel in a conda env.
Thank you!

**Comment by diazandr3s** [comment]
Hi @AgustinSaturni,

> Im alredy labeled a few CTs and I want to train the model but Im using a notebook with an integrated gpu and I would like to use my CPU to go throug all the proces.

We recommend using the GPU when working with MONAI Label

Did you manage to start the MONAI Label server? How are you connecting to it? Are you using Slicer?
